### PR TITLE
Remove preview language from evals docs and expand supported evaluators

### PR DESCRIPTION
**Pull request by schnecle**

### CODE REVIEW

File: docs/evaluation.md [hunk 1]
```diff
@@ -57,6 +53,24 @@ genkit eval:flow bobQA --input testQuestions.json --output eval-result.json
 Note: Below you can see an example of how an LLM can help you generate the test
 cases.
 
+## Supported Evaluator Plugins
```
Comment (kevinthecheung) [inline docs/evaluation.md:56]:
```suggestion
## Supported evaluators
```

File: docs/evaluation.md [hunk 2]
```diff
@@ -57,6 +53,24 @@ genkit eval:flow bobQA --input testQuestions.json --output eval-result.json
...
+- Answer Relevancy
+- Maliciousness
+
+### VertexAI Rapid Evaluators
+
+We support a handful of VertexAI Rapid Evaluators via the [VertexAI Plugin](/docs/plugins/vertex-ai#evaluation).
+
+### Langchain Evaluators
+
+Firebase Genkit supports [Langchain Criteria Evaluation](https://python.langchain.com/docs/guides/productionization/evaluation/string/criteria_eval_chain/) via the Langchain Plugin.
```
Comment (kevinthecheung) [inline docs/evaluation.md:72]:
```suggestion
### Genkit evaluators

Genkit includes a small number of native evaluators, inspired by RAGAS, to help you get started:

- Faithfulness
- Answer Relevancy
- Maliciousness

### Evaluator plugins

Genkit supports additional evaluators through plugins:

- VertexAI Rapid Evaluators via the [VertexAI Plugin](plugins/vertex-ai#evaluation).
- [LangChain Criteria Evaluation](https://python.langchain.com/docs/guides/productionization/evaluation/string/criteria_eval_chain/) via the [LangChain plugin](plugins/langchain.md).
```

**Review by kevinthecheung — CHANGES_REQUESTED** [review]

File: docs/evaluation.md (hunk 1 above)
Comment (schnecle) [reply docs/evaluation.md]: done.

File: docs/evaluation.md (hunk 2 above)
Comment (schnecle) [reply docs/evaluation.md]: done.

**Review by pavelgj — APPROVED** [review]

**Review by ssbushi — APPROVED** [review]

**Review by tagboola — APPROVED** [review]

**Review by kevinthecheung — APPROVED** [review]

**Pull request merged**